### ISSUE TITLE
Determine what to do with asides

### ISSUE THREAD

**Issue by techwiz24** [techwiz24/EECS2500#26]
Some lines contain asides or comments that the characters make. These are indicated by `--`, are not always surrounded by spaces. For example:

```
Dies

HAMLET 
Heaven make thee free of it! I follow thee.
I am dead, Horatio. Wretched queen, adieu!
You that look pale and tremble at this chance,
That are but mutes or audience to this act,
Had I but time--as this fell sergeant, death,
Is strict in his arrest--O, I could tell you--
But let it be. Horatio, I am dead;
Thou livest; report me and my cause aright
To the unsatisfied.
```

As it exists now, the parser will treat `time--as` and `arrest--O` as one word. I think `--` should also indicate a separate word. Need to check with instructor.

This would explain why certain tools give different word counts

**Comment by techwiz24** [comment]
Closed in a8a0076

**Comment by techwiz24** [comment]
Actually, `time--as` ***is*** one lone, weird word according to the professor...
